WTWSMS/common/dynasties/00_dynasties.txt

```diff
@@ -28130,7 +28130,7 @@
 	culture = old_slavic
 }
 1042167 = {
-	name="Atillid" 
+	name="the Hun" 
 	culture = hunnic 
```

Comment:
Unlike the boring, conjectural, and generic "Attilid", the nickname "the Hun" was a contemporary term used for Attila and his family during Attila's lifetime.

Unlike Genghis Khan, whose last name was Borjigin, we have no source on Attila's surname. House "the Hun" was the dynasty encoded until you mistakenly turned it into a conjectural, generic surname. Right now, if I had an illegitimate child as a Hunnic character, HE could get an "Attilid" dynasty as well! That's insane; Attila the Hun should have an unique dynasty befitting his prestige.

Once again, as there is no known surname for Attila the Hun, we should stick with what sounds better and was more used in the time period than a generic suffix. Attila and family were referred to as "the Hun" while Attila was alive. The change should stay until we find proof of a real surname.

Didn't Enlil choose House "the Hun" before you changed it?